### PR TITLE
[TACHYON-1106] Follow up to rename some String constants and sort properties

### PR DESCRIPTION
**Pull request by apc999**
Real changes are constants renaming in `common/src/main/java/tachyon/Constants.java` and property sorting in `common/src/main/resources/tachyon-default.properties`

Other changes are done by IDE refactoring.

### CODE REVIEW

**Comment by apc999** [comment]
Jenkins, test this please.

File: common/src/main/java/tachyon/Constants.java [hunk 1]
```diff
@@ -98,41 +98,40 @@
...
   public static final String UNDERFS_GLUSTERFS_MOUNTS = "tachyon.underfs.glusterfs.mounts";
   public static final String UNDERFS_GLUSTERFS_MR_DIR =
       "tachyon.underfs.glusterfs.mapred.system.dir";
-  public static final String USE_ZOOKEEPER = "tachyon.zookeeper.enabled";
+  public static final String ZOOKEEPER_ENABLED = "tachyon.zookeeper.enabled";
   public static final String ZOOKEEPER_ADDRESS = "tachyon.zookeeper.address";
   public static final String ZOOKEEPER_ELECTION_PATH = "tachyon.zookeeper.election.path";
   public static final String ZOOKEEPER_LEADER_PATH = "tachyon.zookeeper.leader.path";
-  public static final String UNDERFS_HADOOP_PREFIXS = "tachyon.underfs.hdfs.prefixes";
+  public static final String UNDERFS_HDFS_PREFIXS = "tachyon.underfs.hdfs.prefixes";
```
Comment (yupeng9) [inline common/src/main/java/tachyon/Constants.java]: move this up to UNDERFS group?

File: common/src/main/resources/tachyon-default.properties [hunk 2]
```diff
@@ -47,84 +47,84 @@ tachyon.integration.mesos.jre.version=jre1.7.0_79
...
 tachyon.master.journal.formatter.class=tachyon.master.journal.JsonJournalFormatter
-tachyon.master.journal.tailer.sleep.time.ms=1000
-tachyon.master.journal.tailer.shutdown.quiet.wait.time.ms=5000
 tachyon.master.journal.log.size.bytes.max=10MB
-tachyon.master.format.file_prefix=_format_
-tachyon.master.bind.host=0.0.0.0
+tachyon.master.journal.tailer.shutdown.quiet.wait.time.ms=5000
+tachyon.master.journal.tailer.sleep.time.ms=1000
+tachyon.master.lineage.checkpoint.interval.ms=1200000
+tachyon.master.lineage.checkpoint.strategy.class=tachyon.master.lineage.checkpoint.CheckpointLatestScheduler
```
Comment (yupeng9) [inline common/src/main/resources/tachyon-default.properties]: rename this to be consistent?

File: common/src/main/resources/tachyon-default.properties [hunk 3]
```diff
@@ -47,84 +47,84 @@ tachyon.integration.mesos.jre.version=jre1.7.0_79
...
+tachyon.worker.data.port=29999
 tachyon.worker.data.server.class=tachyon.worker.netty.NettyDataServer
+tachyon.worker.evictor.class=tachyon.worker.block.evictor.LRUEvictor
+tachyon.worker.evictor.lrfu.attenuation.factor=2.0
+tachyon.worker.evictor.lrfu.step.factor=0.25
+tachyon.worker.lineage.heartbeat.interval.ms=600000
+tachyon.worker.memory.size=128MB
 tachyon.worker.network.netty.boss.threads=1
-tachyon.worker.network.netty.worker.threads=0
+tachyon.worker.network.netty.file.transfer=MAPPED
```
Comment (yupeng9) [inline common/src/main/resources/tachyon-default.properties]: Just curious, we don't need to update the web docs?

Comment (apc999) [inline common/src/main/resources/tachyon-default.properties]: done in 0.8-docs branch

File: common/src/main/resources/tachyon-default.properties (hunk 2 above)
Comment (apc999) [inline common/src/main/resources/tachyon-default.properties]: done

File: common/src/main/java/tachyon/Constants.java (hunk 1 above)
Comment (apc999) [inline common/src/main/java/tachyon/Constants.java]: done

**Comment by yupeng9** [comment]
LGTM. Can you resolve the conflicts?

**Comment by apc999** [comment]
conflicts resolved. Thanks

**Comment by apc999** [comment]
Jenkins, test this please.

File: common/src/main/java/tachyon/Constants.java [hunk 4]
```diff
@@ -261,20 +260,24 @@
...
+  public static final String USER_FILE_TACHYON_STORAGE_TYPE_DEFAULT =
       "tachyon.user.file.tachyonstoragetype.default";
-  public static final String USER_DEFAULT_UNDER_STORAGE_TYPE =
+  public static final String USER_FILE_UNDER_STORAGE_TYPE_DEFAULT =
       "tachyon.user.file.understoragetype.default";
-  public static final String USER_REMOTE_BLOCK_READER = "tachyon.user.block.remote.reader.class";
-  public static final String USER_REMOTE_BLOCK_WRITER = "tachyon.user.block.remote.writer.class";
+  public static final String USER_BLOCK_REMOTE_BLOCK_READER =
+      "tachyon.user.block.remote.reader.class";
+  public static final String USER_BLOCK_REMOTE_BLOCK_WRITER =
```
Comment (calvinjia) [inline common/src/main/java/tachyon/Constants.java]: This feels kind of repetitive? (two `BLOCK`s)

Comment (apc999) [inline common/src/main/java/tachyon/Constants.java]: done

**Comment by apc999** [comment]
@calvinjia , comments addressed. PTAL

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**